compute_endpoint/globus_compute_endpoint/cli.py

```diff
@@ -369,6 +373,11 @@ def read_config(endpoint_dir: pathlib.Path) -> Config:
         if not config:
             raise Exception(f"Unable to import configuration (no config): {conf_path}")
         spec.loader.exec_module(config)
+
+        # Possible display_name entry is missing.
+        config.config.display_name = None
+        if hasattr(config, "meta"):
+            config.config.display_name = config.meta.get("display_name", None)
```

Comment:
With the most recent commit, this is now stale, yeah?

Reply:
Removed.  Oops